src/main/resources/com/cloudbees/jenkins/support/config/SupportAutomatedBundleConfiguration/components.jelly

```diff
@@ -0,0 +1,26 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" 
+         xmlns:f="/lib/form">
+    <j:forEach var="component" items="${instance.applicableComponents}">
+        <f:entry field="component">
+            <div name="components">
```

Comment:
Shouldn't this be in an upper level?

Reply:
This is very similar to what we have in the `SupportAction`. I just wanted to split the "components" part in a jelly file.

Note: IMO we should create an improvement ticket about creating a reusable jelly component selector. And improve the UX as well.

Reply:
If it works fine, I'm OK at this point.